app/assets/javascripts/analytics/custom-dimensions.js

```diff
@@ -47,6 +47,7 @@
       'navigation-legacy': {dimension: 30, defaultValue: 'none'},
       'navigation-page-type': {dimension: 32, defaultValue: 'none'},
       'political-status': {dimension: 7},
+      'publishing-app': {dimension: 89},
```

Comment:
I don't disagree with adding this, but I'm surprised that dimensions like `format` and `schema-name` don't separate Smart Answer pages from Transactions.

Reply:
@cbaines : smart-answers are an unusual case as their start page is being rendered as a transaction. The effect of this is that they are categorised as having a format and schema-name of transaction in GA. 

ie: https://www.gov.uk/api/content/marriage-abroad